commitizen/cz/conventional_commits/conventional_commits.py

```diff
@@ -45,54 +45,76 @@ def questions(self) -> List[Dict[str, Any]]:
                 "type": "list",
                 "name": "prefix",
                 "message": "Select the type of change you are committing",
+                "use_shortcuts": self.config.settings['use_shortcuts'],
                 "choices": [
                     {
                         "value": "fix",
                         "name": "fix: A bug fix. Correlates with PATCH in SemVer",
+                        "key": "x",
                     },
                     {
                         "value": "feat",
                         "name": "feat: A new feature. Correlates with MINOR in SemVer",
+                        "key": "f",
+                    },
+                    {
+                        "value": "docs",
+                        "name": "docs: Documentation only changes",
+                        "key": "d",
                     },
-                    {"value": "docs", "name": "docs: Documentation only changes"},
                     {
                         "value": "style",
                         "name": (
                             "style: Changes that do not affect the "
                             "meaning of the code (white-space, formatting,"
                             " missing semi-colons, etc)"
                         ),
+                        "key": "s",
                     },
                     {
                         "value": "refactor",
                         "name": (
                             "refactor: A code change that neither fixes "
                             "a bug nor adds a feature"
                         ),
+                        "key": "r",
                     },
                     {
                         "value": "perf",
                         "name": "perf: A code change that improves performance",
+                        "key": "p",
                     },
                     {
                         "value": "test",
                         "name": (
                             "test: Adding missing or correcting " "existing tests"
                         ),
+                        "key": "t",
                     },
                     {
                         "value": "build",
                         "name": (
                             "build: Changes that affect the build system or "
                             "external dependencies (example scopes: pip, docker, npm)"
                         ),
+                        "key": "b",
                     },
                     {
                         "value": "ci",
                         "name": (
                             "ci: Changes to our CI configuration files and "
                             "scripts (example scopes: GitLabCI)"
                         ),
+                        "key": "c",
+                    },
+                    {
+                        "value": "chore",
```

Comment:
We decide not to add "chore" as this is not required by conventional commit https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type